src/crosshook-native/src/styles/proton-manager.css

```
@@ -3,18 +3,82 @@
+  border-radius: var(--crosshook-radius-md);
+  background:
+    linear-gradient(180deg, rgba(59, 130, 246, 0.08), rgba(15, 23, 42, 0.14)),
+    var(--crosshook-color-bg-elevated);
+}
+
+.crosshook-proton-manager-page__path-value {
+  max-width: 32rem;
+  font-family: var(--crosshook-font-mono);
+  font-size: 0.82rem;
+  line-height: 1.5;
+  word-break: break-word;
```

Stylelint (17.9.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)